scripts/setup-olm.sh

```diff
@@ -76,14 +77,22 @@ then
     $OPERATOR_SDK olm uninstall || true
 fi
 
+COUNT=$(kubectl get namespace openshift-operator-lifecycle-manager -o yaml 2>/dev/null | grep config.openshift.io/v1 | wc -l)
+
+if [ $COUNT -gt 0 ]
+then
+    OLM_NS=openshift-marketplace
+    OLM_OP_NS=openshift-operator-lifecycle-manager
+fi
+
 # Setup olm if not already present
-if ! kubectl get -n olm deployment olm-operator
+if ! kubectl get -n $OLM_OP_NS deployment olm-operator
```

Comment:
I think it would be more clear if we just skipped this entire if block if it is openshift.  That way we don't need the OLM_OP_NS variable.

```
if ! $SCRIPT_DIR/is-openshift.sh && ! kubectl get -n $OLM_NS deployment olm-operator
then
...
```

Reply:
I will address both in my next commit